lib/cluster.js

```diff
@@ -31,14 +31,12 @@ function Worker(options) {
   this.exitedAfterDisconnect = undefined;
 
   Object.defineProperty(this, 'suicide', {
-    get: internalUtil.deprecate(() => {
```

Comment:
I would describe this as `cluster: better suicide deprecation message` (or something of the like), since its a change to improve the deprecation message, along with a unit test for that change.